src/client/kube_store.test.ts

```diff
@@ -54,3 +56,24 @@ test('Test NamespacedResource setResource', () => {
   store.setResource(migResource, planName, examplePlan);
   expect(_.isEqual(expected, store.db)).toBe(true);
 });
+
+test('Test NamespacedResource patchResource', () => {
+  const migResource = new MigResource(MigResourceKind.MigCluster, testNs);
+  const store = new KubeStore('_host');
+  const patch = {'test':'foo'};
+
+  const expected = {
+
+      namespace: testNs,
+        _gvk:
+          { group: 'migration.openshift.io',
+            version: 'v1alpha1',
+            kindPlural: 'migclusters' },
+          test: 'foo',
+    };
+
+  const patchedObject = store.patchResource(migResource, clusterName, patch);
+  localStorage.clear();
+  localStorage.setItem('test', JSON.stringify(patchedObject))
+  expect(patchedObject).toEqual(expected);
```

Comment:
We want to test how the patch command works.
So the workflow would be something like:

1) Determine some piece of data that already exists in kube_store
2) Create a patch command to modify that existing data
3) Issue the patch command
4) Check the data in kube_store and verify it was modified as you expected.